### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by gonzalon**
I add the implementation of `get_stats` and `get_user_stats`.
Also I found that every time the code was updated I need to change the code for the comments, I use instabot with the comments in Spanish. So I made a change for being able to set the comment values to the bot from outside.
I also add the option for custom words for a tag in particular.

There is a example of how it would work in the `example.py` file. But mainly this is compatible with the previous versions, so if you are not going to use this feature you don't need to change your code at all.

### CODE REVIEW

**Comment by LevPasha** [comment]
@gonzalon thx! Watch it soon!

File: example.py [hunk 1]
```diff
@@ -3,10 +3,10 @@
 
 from instabot import InstaBot
 
-bot = InstaBot(login="my_username", password="my_password",
+bot = InstaBot(login="robotitogram", password="gonzal0",
```
Comment (LevPasha) [inline example.py]: hmmm, look like you should hide this...

**Comment by LevPasha** [comment]
@gonzalon watch this https://github.com/LevPasha/instabot.py/issues/91

**Comment by gonzalon** [comment]
Ups, thanks, changed the password. Anyway it's just a test account :+1:

**Comment by nivong** [comment]
@LevPasha why isn't this merged yet?

**Comment by gonzalon** [comment]
@nivong don't really know :P I have been using it without problems so far. Have you been able to test it?

**Comment by LevPasha** [comment]
@nivong @gonzalon ammmm, i'm slowpoke)
Problem is - i have no time to test it, and i don't want to merge code without test...

**Comment by LevPasha** [comment]
Boys and girls (ahaha), please test it, and i will merge!

**Comment by polaroi8d** [comment]
Hey @gonzalon sorry for the inconvenience, can you solve the conflict? After it I will test your patch and merge if its works right.

**Comment by gonzalon** [comment]
@polaroi8d Sorry the delay!! Have been busy, still need any help? Let me know, cheers!

**Comment by polaroi8d** [comment]
@gonzalon  I just want to pulled to the master, but there is a conflict know.

**Pull request closed without merging**